pkg/kubeclient/kubeclient.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023 The Kubernetes Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package kubeclient contains common CLI flags for creating a Kubernetes client.
+package kubeclient
```

Comment:
It feels as if we should have a `flags` package here, with files for kubeclient.go and nodeallocationstate.go, rather than two separate packages that only deal with flags but have no indication of that in the package name.

Reply:
A single `pkg/flags` also works.